python/snewpy/flavor.py

```diff
@@ -58,6 +58,41 @@ def to_tex(self):
         if self.is_muon or self.is_tauon:
             lepton = '\\'+lepton
         return f"${base}_{{{lepton}}}$"
+
+    def to_marley(self):
+        """The neutrino name in the Marley format. 
+           See Table 3 in https://arxiv.org/pdf/2101.11867
+           The name for the sterile is a SNEWPY extension
+        """
+        name = "v"
+        if self.is_electron:
+            name += "e"
+        elif self.is_muon:
+            name += "u"
+        elif self.is_tauon:
+            name += "t"
+        else: 
+            name += "s"            
+        if self.is_antineutrino:
+            name += "bar"
+        return name    
+
```

Comment:
Probably better to avoid having this in core snewpy and move it to the script that transforms snewpy fluences into MARLEY’s JSON format; to avoid bugs if MARLEY changes down the road.